news/index.md

```diff
@@ -1,58 +1,95 @@
 # News
 
+## Version 1.1 of the Oxford Common File Layout (OCFL) Released
+
+**7 October 2022**
+
+The OCFL Editors are pleased to announce version 1.1 of the Oxford Common File Layout. The focus of this revision
+is correction and clarification based on implementation experience and community feedback. Version 1.1 is backwards
+compatible with version 1.0.
+
+### What new information is available?
+
+The OCFL website at https://ocfl.io, includes the most up to date version of the specification and the implementation
+notes as well as the latest editors draft.
+
+The [OCFL Specification v1.1](https://ocfl.io/1.1/spec/) defines both OCFL Objects, a simple structure for content and
+a JSON document (`inventory.json`) which provides a straightforward but comprehensive register for the object and
+versions of its content, and an OCFL Storage Root, an arrangement for how OCFL Objects are laid out on physical storage.
+It also contains examples illustrating the use of the OCFL, and explanations that ground decisions in prior experience.
+
+There is an accompanying [Change Log](https://ocfl.io/1.1/spec/change-log.html) that details the changes from version 1.0
+to version 1.1. It is designed to assist implementers with updates to their implementations.
+
+The companion [OCFL Implementation Notes v1.1](https://ocfl.io/1.1/implementation-notes/) contains advice for implementing
+the specification including recommendations for digital preservation, storage handling, client behaviors, and best
+practices for dealing with OCFL Objects in motion.
+
+The OCFL Editors are also releasing updated [validation rules](https://ocfl.io/1.1/spec/validation-codes.html) and
+additional [fixture objects](https://github.com/OCFL/fixtures) for testing OCFL implementations against the specification. We
```

Comment:
We maintain version-specific fixture updates: https://github.com/OCFL/fixtures/tree/main/1.1
In the case that we do not want to actually "release" the fixtures, it may make sense to link to the 1.1 directory above.

Reply:
I thought about this and concluded that it is best to keep to the overall fixtures reference. One needs both 1.0. and 1.1 fixtures to properly validate a 1.1 implementation.